illumina_copyNumber.pm

```diff
@@ -19,14 +20,24 @@ sub parseSamples {
 			print "WARNING: $sample is not passing copy number samplename parsing, skipping \n\n";
 			next;
 		}
+		if($opt{SET_HAS_METADATA} eq "yes"){
+		    print "Reading Tumour/Reference names from metadata\n\n";
+		    my $metadata = metadataParse($opt{OUTPUT_DIR});
+		    # Reference sample
```

Comment:
I would argue that the clarity offered so people are aware that the metadata is being read is worth the extra comment line, most end users of the pipeline will not be going through the code.